<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHallDialog</class>
 <widget class="QWidget" name="AddHallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Зал</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="cinema_label">
        <property name="text">
         <string>Кинотеатр</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="count_row_label">
        <property name="text">
         <string>Количество рядов</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="count_col_label">
        <property name="text">
         <string>Количество колонок</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cinema_combo_box"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="name_line_edit"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="count_row_edit"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="count_col_elit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
